src/material-experimental/mdc-slider/_slider-theme.scss

```diff
@@ -51,3 +71,38 @@
   }
 }
 
+@mixin custom-slider-color($color) {
```

Comment:
I assume we don't want people using this directly? in that case we should name it `mat-private-...`

Reply:
Since it's only used in this file, it can be prefixed with an underscore which will prevent it from being exported under the new Sass module system.

Reply:
Ah good to know, I don't think I ever quite understood when it was ok to use underscore prefixes